src/index.html

```diff
@@ -6,13 +6,182 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Weather App</title>
     <link rel="stylesheet" href="main.bundle.css" />
+    <style>
+      .container {
+        font-size: 10px;
+        max-width: 800px;
+        margin: 5vh auto 0;
+        position: relative;
+        width: 100%;
+      }
+      .main--picture {
+        position: relative;
+      }
+      .main--picture__image {
+        min-height: 330px;
+        min-width: 480px;
+        width: 60%;
+        height: 30%;
+        margin: 0 auto;
+        display: block;
+        align-items: center;
+      }
+      .main--picture__text {
+        font-size: 2.8rem;
+        font-weight: bolder;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        opacity: 0.8;
+      }
+      .search {
+        width: 100%;
+        margin: 20px auto;
+      }
+      .search__input {
+        display: block;
+        width: 90%;
+        margin: 0 auto;
+        padding: 10px;
+        border-radius: 10px;
+        border: 2px solid blue;
+        font-size: 1.5rem;
+        box-shadow: 0 0 10px rgba(1, 1, 1, 0.2);
+      }
+      .places {
+        flex-wrap: nowrap;
+        display: flex;
+      }
+      .places__box {
+        flex-grow: 1;
+        width: 22%;
+        padding: 0 15px;
+        cursor: pointer;
+      }
+      .box__image {
+        width: 100%;
+        height: 160px;
+        display: block;
+        object-fit: cover;
+        border-radius: 10px;
+      }
+      .box__text {
+        text-align: center;
+        font-size: 1.25rem;
+        color: black;
+      }
+      .hidden {
+        display: none;
+      }
+    </style>
+    <style class="RWD">
+      @import "variables";
```

Comment:
importowanie styli to funkcjonalnośc sass, nie można tego robić, w html, sugestia dotyczy błędu który widać w konsoli